app/server/tests/get_community.test.js

```diff
@@ -0,0 +1,121 @@
+const endpoint = 'https://hwp-express-api-d63404-dev.apps.silver.devops.gov.bc.ca/api';
```

Comment:
We need to get this URL from an environment variable please. We cannot hard code URLs like this into our source.

Reply:
Also, file naming convention needs to follow the naming convention that Dylan, Brady, Zach and Brendan have set. I believe it's likeThis. Please confirm with the files that Dylan has already committed to the repo.

Reply:
will update the file name and try to get the environment variables 👍

Reply:
yes we are using camelCase